helm/idol-nifi/values.yaml

```diff
@@ -0,0 +1,43 @@
+# autoscalingEnabled - deploy a horizontal pod autoscaler for the nifi statefulset
+autoScalingEnabled: true
+# autoscalingMinReplicas - the minimum size of the nifi statefulset
+autoScalingMinReplicas: 1
+# autoscalingMaxReplicas - the maximum size of the nifi statefulset
+autoScalingMaxReplicas: 8
+
+# idolImageRegistry - used to construct container image name: <idolImageRegistry>/<nifiImage>:<idolVersion>
+idolImageRegistry: microfocusidolserver
+# idolVersion - used to construct container image name: <idolImageRegistry>/<nifiImage>:<idolVersion>
+idolVersion: "23.4"
+
+# imagePullSecrets - secrets used to pull container images
+imagePullSecrets:
+- "dockerhub-secret"
+
+nifi:
```

Comment:
Do we have a good reason to use nested over flat here (cf. [Best Practices](https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values), _"In most cases, flat should be favored over nested."_)?

TBH I think I prefer the structure of nested, but we deliberately flattened (most) values in distributed-idol to comply with the above advice.

Reply:
> When there are a large number of related variables, and at least one of them is non-optional, nested values may be used to improve readability.

I much prefer structure too and figurem can justify readability winning out. 

> For optimal safety, a nested value must be checked at every level
I think mostly this is mitigated by having default values, which ensures the structure?

Reply:
Fair, that's a given exception to the rule. ✔️